apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-container-contentlet.service.ts

```diff
@@ -20,11 +21,12 @@ export class DotContainerContentletService {
      */
     getContentletToContainer(
         container: DotPageContainer,
-        content: DotPageContent
+        content: DotPageContent,
+        page: DotPage
```

Comment:
pass only the `inode` ?

Reply:
I think is better pass all the page, because in this way it is its explained what the method need better